src/dev/impl/DevToys/Views/Tools/Converters/NumberBaseConverter/NumberBaseConverterToolPage.xaml

```diff
@@ -26,14 +32,30 @@
                     Style="{StaticResource RightAlignedToggleSwitchStyle}"
                     IsOn="{x:Bind ViewModel.IsFormatted, Mode=TwoWay, UpdateSourceTrigger=PropertyChanged}"/>
             </controls:ExpandableSettingControl>
+            <controls:ExpandableSettingControl
+                Title="{x:Bind ViewModel.Strings.AdvancedModeLabel}">
+                <controls:ExpandableSettingControl.Icon>
+                    <FontIcon Glyph="&#xF7B2;" />
+                </controls:ExpandableSettingControl.Icon>
+                <ToggleSwitch
+                    Style="{StaticResource RightAlignedToggleSwitchStyle}"
+                    IsOn="{x:Bind ViewModel.AdvancedMode, Mode=TwoWay, UpdateSourceTrigger=PropertyChanged}"/>
+            </controls:ExpandableSettingControl>
         </StackPanel>
 
         <muxc:InfoBar 
             IsOpen="{x:Bind ViewModel.IsInfoBarOpen, Mode=OneWay}" 
             Severity="Warning" 
             Message="{x:Bind ViewModel.InfoBarMessage, Mode=OneWay}" />
 
-        <controls:CustomTextBox
+        <Grid Visibility="{x:Bind ViewModel.AdvancedMode, Mode=OneWay, Converter={StaticResource InvertedBooleanToVisibilityConverter}}">
+            <nbc:BasicNumberBaseConverterControl/>
```

Comment:
No need to place this control in a Grid. Just place the Visibility binding directly on the control.